### ISSUE TITLE
HPRC Dataset should be dropped from anvil3 

### ISSUE THREAD

**Issue by bvizzier-ucsc** [DataBiosphere/azul#5690]
Earlier today I was looking at https://prod.anvil.gi.ucsc.edu/explore/datasets and noticed that [ANVIL_HPRC](https://prod.anvil.gi.ucsc.edu/explore/datasets/6a15c882-2eb5-41af-9bf6-00b9ccdfad22) was included. 

This dataset has a number of temporary files in it should not have, resulting in a large number of file formats showing up in the Data Explorer (e.g., 9583 formats).  The HPRC team is in the process of cleaning them up.

It was my understanding that we were going to drop this dataset from the index until it is cleaned up.   

This ticket is to request the dropping of the ANVIL_HPRC dataset until the file issue is resolved.

- [x] Security design review completed; the Resolution of this issue does **not** …
  - [x] … affect authentication; for example:
    - OAuth 2.0 with the application (API or Swagger UI)
    - Authentication of developers with Google Cloud APIs
    - Authentication of developers with AWS APIs
    - Authentication with a GitLab instance in the system
    - Password and 2FA authentication with GitHub
    - API access token authentication with GitHub
    - Authentication with 
  - [x] … affect the permissions of internal users like access to
    - Cloud resources on AWS and GCP
    - GitLab repositories, projects and groups, administration
    - an EC2 instance via SSH
    - GitHub issues, pull requests, commits, commit statuses, wikis, repositories, organizations
  - [x] … affect the permissions of external users like access to
    - TDR snapshots
  - [x] … affect permissions of service or bot accounts
    - Cloud resources on AWS and GCP
  - [x] … affect audit logging in the system, like
    - adding, removing or changing a log message that represents an auditable event
    - changing the routing of log messages through the system
  - [x] … affect monitoring of the system
  - [x] … introduce a new software dependency like
    - Python packages on PYPI
    - Command-line utilities
    - Docker images
    - Terraform providers
  - [x] … add an interface that exposes sensitive or confidential data at the security boundary
  - [x] … affect the encryption of data at rest
  - [x] … require persistence of sensitive or confidential data that might require encryption at rest
  - [x] … require unencrypted transmission of data within the security boundary
  - [x] … affect the network security layer; for example by 
    - modifying, adding or removing firewall rules
    - modifying, adding or removing security groups
    - changing or adding a port a service, proxy or load balancer listens on
- [x] Documentation on any unchecked boxes is provided in comments below

**Comment by dsotirho-ucsc** [comment]
Related ticket #5546 PR #5559

**Comment by achave11-ucsc** [comment]
@hannes-ucsc: "We dropped this snapshot from `anvil2` because of a unrelated problem with that snapshot. That problem was fixed and the replacement snapshot was added to `anvil3`. This latter snapshot is still affected by another problem that causes an excessive number of entries in the file format term facet. Until that problem is resolved and a replacement snapshot is released, we will remove the snapshot from `anvil3`."
